lib/veri_report.ml

```diff
@@ -0,0 +1,56 @@
+open Core_kernel.Std
+open Bap.Std
+open Bap_traces.Std 
+
+module Map = String.Map
```

Comment:
Also, try to use generic `Map` and `Set` instead of fixed `T.Map` and `T.Set`. The latter two are needed only for constructing.